modules/api/functional_test/live_tests/batch/create_batch_change_test.py

```diff
@@ -3093,6 +3094,58 @@ def test_create_batch_update_record_in_shared_zone_for_unassociated_user_in_owne
             shared_client.wait_until_recordset_change_status(delete_rs, 'Complete')
 
 
+def test_create_batch_with_global_acl_rule_applied_succeeds(shared_zone_test_context):
+    """
+    Test that a user with a relevant global acl rule can update forward and reverse records, regardless of their current ownership
```

Comment:
can you add one simpler negative case.

Namely:

1. Create a record with an owner group set in a zone that is NOT in the global ACL
2. Attempt to delete the record via a batch change using a user that IS in the global ACL
- expected result = fail -

Reply:
added here https://github.com/pauljamescleary/vinyldns/pull/3/files#diff-090b2d00e8c30c6696c34e10ed759ac5R3163